web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -52,12 +60,13 @@
     @ExceptionHandler(SampleNotFoundException.class)
     public ResponseEntity<ErrorResponse> handleSampleNotFound(SampleNotFoundException ex) {
 
-        return new ResponseEntity<>(new ErrorResponse("Sample not found: " + ex.getSampleId()),
-                HttpStatus.NOT_FOUND);
+        return new ResponseEntity<>(new ErrorResponse("Sample not found in study " + ex.getStudyId() + ": " +
+                ex.getSampleId()), HttpStatus.NOT_FOUND);
     }
 
     @ExceptionHandler(MissingServletRequestParameterException.class)
-    public ResponseEntity<ErrorResponse> handleMissingServletRequestParameter(MissingServletRequestParameterException ex) {
+    public ResponseEntity<ErrorResponse> handleMissingServletRequestParameter(
```

Comment:
I use Intellij's vertical line, which is 120 chars, and it barely fits my 13" laptop's screen :)